packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -112,18 +112,23 @@ const Ready = ({
             </div>
           </>
         )}
-        {!isSync && isSignedIn && serviceName && (
-          <FtlMsg id="ready-use-service" vars={{ serviceName }}>
-            <p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>
-          </FtlMsg>
-        )}
-        {!isSync && isSignedIn && !serviceName && (
-          <FtlMsg id="ready-use-service-default">
-            <p className="my-4 text-sm">
-              You’re now ready to use account settings
-            </p>
-          </FtlMsg>
-        )}
+        {!isSync &&
```

Comment:
nit: A little hard to read this with the chained variables. Maybe pulling them into a variable like `showReadyToUseService` and `showReadyToUseSettings`.

Reply:
Very nice! Thanks for the suggestion. 👌